aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -122,7 +122,7 @@ enum class Activation {
   GELU,
 };
 
-#if !defined(USE_ROCM) && !defined(_MSC_VER)
+#if (defined(USE_ROCM) && ROCM_VERSION >= 50600) || (defined(CUDA_VERSION) && CUDA_VERSION >= 11000 && !defined(_MSC_VER))
```

Comment:
The change about `CUDA_VERSION` is a little confusing. Are we going to change CUDA's behavior as well?